jax/_src/numpy/reductions.py

```diff
@@ -1069,9 +1069,44 @@ def _std(a: ArrayLike, axis: Axis = None, dtype: DTypeLike | None = None,
   return lax.sqrt(var(a, axis=axis, dtype=dtype, correction=correction, keepdims=keepdims, where=where))
 
 
-@implements(np.ptp, skip_params=['out'])
 def ptp(a: ArrayLike, axis: Axis = None, out: None = None,
         keepdims: bool = False) -> Array:
+  r"""Return the range of values (maximum - minimum) along a given axis.
```

Comment:
I think this is a bit too similar to the numpy verbiage. How about something like the "peak-to-peak range", or explicitly the "difference between the maximum and minimum values"?

Reply:
Changed